### PR TITLE
Add 3.0 blog post on branching/relasing

### PR DESCRIPTION
**Pull request by bmbouter**

### CODE REVIEW

File: _posts/2019-10-31-pulp-3-GA-update.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+### Release Lead
+
+Each 3.y release will have a release lead who performs the cherry picking of fixes to be included in
+the 3.y.z releases. Brian Bouterse will be the release lead for pulpcore for the 3.0, which includes
+releases 3.0.0, 3.0.1, and any other 3.0 z-stream releases. For 3.1 and its z-stream, another release
+lead will be identified.
+
+Each plugin team is encouraged to have a release lead who performs similar funcitons for that
+plugins y-release.
```
Comment (CodeHeeler) [inline _posts/2019-10-31-pulp-3-GA-update.md]: s/funcitons/functions
s/plugins/plugin's

File: _posts/2019-10-31-pulp-3-GA-update.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+When a release is ready it is tagged as the x.y.z version number. For the 3.0 y-release this will be
+3.0.0. This tagging is expected on Dec 3rd. Once tagged Travis tests things one final time before
+pushing the bits to PyPI and the bindings to PyPI and rubygems.org.
+
+
+### What changed in release 3.y.z?
+
+While the git-history is the ultimate source of truth, it's too large and too much change to easily
+see the meaningful changes that occurred in a release. As such, the changelog for the release is the
+recommended way to see what changed in for a specific release.
```
Comment (CodeHeeler) [inline _posts/2019-10-31-pulp-3-GA-update.md]: "in a specific release" or "for a specific release", either sounds fine, just not both.

File: _posts/2019-10-31-pulp-3-GA-update.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+### Cherry Picking
+
+The z-stream is curated by cherry picking changes from the master branch to the y-release branch,
+e.g. 3.0. The release lead performs the cherry picking as needed and with input from developers and
+users.
+
+
+### Tagging a Release
+
+When a release is ready it is tagged as the x.y.z version number. For the 3.0 y-release this will be
```
Comment (daviddavis) [inline _posts/2019-10-31-pulp-3-GA-update.md]: s/ready/ready,/

File: _posts/2019-10-31-pulp-3-GA-update.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+3.0.0. This tagging is expected on Dec 3rd. Once tagged Travis tests things one final time before
+pushing the bits to PyPI and the bindings to PyPI and rubygems.org.
+
+
+### What changed in release 3.y.z?
+
+While the git-history is the ultimate source of truth, it's too large and too much change to easily
```
Comment (daviddavis) [inline _posts/2019-10-31-pulp-3-GA-update.md]: s/and too much/and has too much/

**Comment by daviddavis** [comment]
This is great. Thanks for writing this up. :+1: 

What's the scope of this blog post? Asking because I don't see anything about how the redmine issues will be handled, how/where the release will be announced, etc

Also, will this information live somewhere more permanently? If not, I wonder about how we will make changes to our release process over time.

File: _posts/2019-10-31-pulp-3-GA-update.md (hunk 1 above)
Comment (bmbouter) [reply _posts/2019-10-31-pulp-3-GA-update.md]: ty fixed in next push.

File: _posts/2019-10-31-pulp-3-GA-update.md (hunk 2 above)
Comment (bmbouter) [reply _posts/2019-10-31-pulp-3-GA-update.md]: lol yeah. fixed in next push, ty.

File: _posts/2019-10-31-pulp-3-GA-update.md (hunk 3 above)
Comment (bmbouter) [reply _posts/2019-10-31-pulp-3-GA-update.md]: fixed in next push, ty.

File: _posts/2019-10-31-pulp-3-GA-update.md (hunk 4 above)
Comment (bmbouter) [reply _posts/2019-10-31-pulp-3-GA-update.md]: ty fixed in next push.

**Comment by bmbouter** [comment]
@daviddavis I ran out of time to add the Redmine section. I'm adding one now.

Also we do need it somewhere more permanent so I filed [this issue](https://pulp.plan.io/issues/5661) and labelled as a GA blocker.

**Review by daviddavis — APPROVED** [review]

**Pull request merged**